qiskit/primitives/base/base_estimator.py

```diff
@@ -192,42 +196,29 @@ def run(
             **run_opts.__dict__,
         )
 
-    @abstractmethod
+    @deprecate_func(since="0.46.0")
```

Comment:
I don't think this will do anything. None of the subclasses of `BaseEstimator` will call `super()._run()` (because then a `NotImplementedError` gets raised) so I'm not sure anything will see this.